### PR TITLE
added CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by dougxc**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+To submit pull requests for mx, you need to sign the [Oracle Contributor Agreement][1].
+There is a white-list for contributors who have signed the OCA so please add a comment
+to your first pull request indicating the name you used to sign the OCA if it isn't clear
+from your github profile.
+
+Pull requests can only be merged by committers and a committer cannot merge
+their own pull requests.
```
Comment (thomaswue) [inline CONTRIBUTING.md]: "their" => "his"

Comment (dougxc) [inline CONTRIBUTING.md]: then I'll go with "his or her"

Comment (thomaswue) [inline CONTRIBUTING.md]: even better :) !

Comment (smarr) [inline CONTRIBUTING.md]: "their" is usually acceptable singular for such situations.

Comment (thomaswue) [inline CONTRIBUTING.md]: OK, I did not know that. My English teacher would have surely objected ;).

Comment (dougxc) [inline CONTRIBUTING.md]: while I'm aware that "their" is acceptable grammar here, I think "his or her" reads better, especially for those whose first language is English ;-)

Comment (dougxc) [inline CONTRIBUTING.md]: btw, is there (or their? ;-) ) a convention for signing off on github pull request reviews?

**Comment by smarr** [comment]
signing off? you mean like this: www.lgtm.in :wink:

**Pull request merged**